src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -23,11 +23,16 @@
     private final RepaymentPlanContent repaymentPlan;
     private final String signerName;
     private final String signerRole;
+    private final boolean admissionResponse;
+    private final boolean partAdmissionResponse;
+
 
     public CCJContent(Map<String, Object> claim,
                       CountyCourtJudgment countyCourtJudgment,
                       LocalDateTime countyCourtJudgmentRequestedAt,
-                      AmountContent amount) {
+                      AmountContent amount,
+                      boolean admissionResponse,
```

Comment:
Do we need two boolean parameters to determine the admission response type? Can't we use `ResponseType` here instead of multiple flags?